YR/ Update FeedOpenCTI_v4 to use the latest PyCTI version(5.12.15) (#32142)
demisto/content: Packs/FeedOpenCTI/ReleaseNotes/2_1_0.md

#### Integrations

##### OpenCTI Feed 4.X

> - **Breaking changes**: These following modifications are compatible with OpenCTI version 5.12.17 and newer, and might not work with older versions.

Reviewer: ```suggestion
**Breaking changes**: The following modifications are compatible with OpenCTI version 5.12.17 and higher, and might not work with older versions.
```

- Updated the Docker image to: *demisto/vendors-sdk:1.0.0.86010*.

> - Updated the *filters* to fit the latest format of the the *pycti* library.

Reviewer 2: ```suggestion
- Improved implementation of the *filters* argument, to match the latest format OpenCTI requires.
```

Author: Done.
